mongodbatlas/resource_mongodbatlas_private_ip_mode.go

```diff
@@ -39,6 +39,7 @@ func resourceMongoDBAtlasPrivateIPMode() *schema.Resource {
 				Required: true,
 			},
 		},
+		DeprecationMessage: "This resource is deprecated, and will be removed in v1.9 release. Please transition to mongodbatlas_cloud_backup_schedule as soon as possible",
```

Comment:
is mongodbatlas_cloud_backup_schedule related to private_ip_mode? if not can you please update

Reply:
![Uploading image.png…]()

Reply:
Removed Please transition to mongodbatlas_cloud_backup_schedule as soon as possible unrelated